### ISSUE TITLE
Error when using guild init with github url in requirements.txt

### ISSUE THREAD

**Issue by FelixAbrahamsson** [guildai/guildai#90]
I'm getting an error when I run `guild init` if I have a github url in my requirements.txt.
Contents of requirements.txt:
```git+https://github.com/Po-Hsun-Su/pytorch-ssim```
Error message from guild init:
```
ERROR: Error [Errno 2] No such file or directory: 'git': 'git' while executing command git clone -q https://github.com/Po-Hsun-Su/pytorch-ssim /tmp/pip-req-build-fd6lz7sl
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?
guild: Command '['/home/aiwizo/Documents/tmp/venv/bin/pip', 'install', '-r', './requirements.txt']' returned non-zero exit status 1.
```

If I simply use an empty requirements.txt for my guild init, then add the github url line and run `pip install -r requirements.txt`, everything works as expected.

**Comment by gar1t** [comment]
Thank you for the report! Looking at this now.

**Comment by gar1t** [comment]
This is resolved and will be available in the next pre-release (0.7.0rc3). Thanks again!
